targeted_test.go

```diff
@@ -44,7 +44,7 @@ func TestParseAnyMatcher(t *testing.T) {
 		{"ab", []byte("a")},
 		{"\u2190\U00001100", []byte("\u2190")},
 		{"\x0d", []byte("\x0d")},
-		{"\xfa", nil},
+		{"\xfa", []byte("\xfa")},
```

Comment:
For me it is unclear, how your PR does influence this test. Do you mind to explain?

Reply:
The anyMatcher invocation itself now will match invalid bytes, if they were parsed and passed to it. The option behavior is gated at the parsing stage, which is bypassed in the test by invoking the matcher directly.

Reply:
:+1: got it, does make sense.